media/editor/state.ts

```diff
@@ -8,13 +8,13 @@ import { FromWebviewMessage, InspectorLocation, MessageHandler, MessageType, Rea
 import { deserializeEdits, serializeEdits } from "../../shared/serialization";
 import { clamp, Range } from "./util";
 
-declare function acquireVsCodeApi(): ({
+const acquireVsCodeApi: () => ({
 	postMessage(msg: unknown): void;
 	getState(): any;
 	setState(value: any): void;
-});
+}) = (globalThis as any).acquireVsCodeApi;
 
-export const vscode = acquireVsCodeApi();
+export const vscode = acquireVsCodeApi?.();
```

Comment:
Linaria evaluates scripts sometimes for some reason (??) and making this changed fixed that evaluation failing